lib/vmpooler/janitor.rb

```diff
@@ -0,0 +1,39 @@
+module Vmpooler
+  class Janitor
+    def initialize
+      # Load the configuration file
+      config_file = File.expand_path('vmpooler.yaml')
+      $config = YAML.load_file(config_file)
```

Comment:
This code is repeated a few times throughout the app. Perhaps it would be better suited in its own class or helper method?